src/lib/addXP.ts

```diff
@@ -45,6 +46,13 @@ async function onMax(user: MUser) {
 	kUser.send(MAXING_MESSAGE).catch(noOp);
 }
 
+async function onBigMileStone(user: MUser, mileStone: number) {
+	const str = `🎉 ${user.badgedUsername}'s minion just achieved ${mileStone / 1_000_000} M XP in every skill!🎉`;
```

Comment:
use `toKMB()`, and rename this function to something more specific like "allSkillXPMilestone"